create-leo-app/template-react-managed-worker/tests/example.spec.js

```diff
@@ -0,0 +1,38 @@
+// @ts-check
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('http://localhost:5173/');
+  page.on('dialog', dialog => dialog.accept());
+});
+
+test('builds', async ({ page }) => {
+    const accountButton = await page.getByTestId("generate-account-button");
+    await expect(accountButton).toHaveText("Click to generate account");
+    await page.waitForTimeout(1000);
```

Comment:
Would prefer not to have any timeouts. Should be waiting for something specific.